Reportedly/Application/Localizations/en.lproj/Localizable.strings

```diff
@@ -42,4 +42,8 @@
 
 "module.Home.MenuLogoutOption" = "Logout";
 
-"module.Loading.Message" = "Loading...";
+"module.Notifications.DailyReportReminderTitle" = "Daily Standup Report";
```

Comment:
I have just found out that we use camel case for domain name like `module.notifications.dailyReportReminderTitle`. Could you please double check all text? 🙏

Reply:
yeah sure, let me update the localization 🙏